pdr_backend/subgraph/test/test_subgraph_predictions.py

```diff
@@ -117,19 +116,20 @@ def test_fetch_filtered_predictions(mock_query_subgraph):
     predictions = fetch_filtered_predictions(
         start_ts=1622547000,
         end_ts=1622548800,
+        first=1000,
+        skip=0,
         filters=["0x18f54cc21b7a2fdd011bea06bba7801b280e3151"],
         network="mainnet",
-        filter_mode=FilterMode.PREDICTOOR,
     )
 
-    assert len(predictions) == 2000
+    assert len(predictions) == 1000
     assert isinstance(predictions[0], Prediction)
     assert predictions[0].user == "0xd2a24cb4ff2584bad80ff5f109034a891c3d88dd"
     assert predictions[0].pair == "ADA/USDT"
     assert predictions[0].address[0] == "0x18f54cc21b7a2fdd011bea06bba7801b280e3151"
     assert predictions[0].trueval is False
     assert predictions[0].prediction is True
-    assert mock_query_subgraph.call_count == 3
+    assert mock_query_subgraph.call_count == 1
```

Comment:
It needs to be designed according to a multiple subgraph call scenario. 
There are mock data for three calls a few lines above. 
If we only test it for one subgraph query call, it won't be the proper test.

Reply:
Inside the subgraph now it's just 1 call so this in right, but in the table it should check that subgraph it's fetched multiple time, I'm just not sure how to mock that. I could just let the function query the subgraph directly and adjust the time inside a mocked ppss